### DISCUSSION TITLE
Failed to Load schema error

### DISCUSSION THREAD

**Discussion by ariba932** [supabase/supabase#31052] [Questions]
Please can someone help why will I have the access to the schema refused.  based error below.....  

I have only one active project ... 

Failed to load schemas
Error: connect ECONNREFUSED 2a05:d01c:30c:9d00:99dc:d204:5b7c:681d:5432

Reload schemas

New table
Search tables
Search tables...

Failed to retrieve tables
Error: connect ECONNREFUSED 2a05:d01c:30c:9d00:99dc:d204:5b7c:681d:5432

Try refreshing your browser, but if the issue persists, please reach out to us via support.

[Contact support](https://supabase.com/dashboard/support/new?category=dashboard_bug&subject=Failed%20to%20retrieve%20tables&error=connect%20ECONNREFUSED%202a05:d01c:30c:9d00:99dc:d204:5b7c:681d:5432)

Below showing resources bursting sometime without not activities from the Apps to the db -- Just curious what is using the the resources.  Although support team not responding to ticket because the project is on free version... 

Same time, I am curious on app failing without proper justification or my ignorance.. thus support call..

![image](https://github.com/user-attachments/assets/303f759e-dca1-4e26-8a16-f897dbb1faff)

**Comment by GaryAustin1** [comment]
You have not provided info on your call or if this is a custom schema.
